platform_plugin_forum_email_notifier/tasks.py

```diff
@@ -63,3 +65,45 @@ def send_email_notification(
             "thread_type": thread_type,
         },
     )
+
+
+@shared_task
+@set_code_owner_attribute
+def notify_users(thread_id, discussion, course_id, body, title, url, object_type):
+    """
+    Get the subscribers for a thread and notify them.
+    """
+    if object_type == ForumObject.THREAD:
+        subscribers = get_subscribers(thread_id)
+    elif object_type in (ForumObject.RESPONSE, ForumObject.COMMENT):
+        subscribers = get_subscribers(discussion.get("id"))
+    else:
+        raise ValueError(f"Invalid thread event type: {object_type}")
+
+    staff_subscribers = get_staff_subscribers(course_id)
```

Comment:
what's the difference between getting subscribers and staff subscribers?

Reply:
A subscriber is a user(staff or course staff) who is subscribed to a thread.

An staff subscriber is a course staff member that configured it's preference for the course to get notified of all posts